### PR TITLE
review for code

### PR DESCRIPTION
**Pull request by naruto08060119487**
reviewing assessment c#

### CODE REVIEW

File: CMP1903_A1_2324/Die.cs [hunk 1]
```diff
@@ -13,10 +13,15 @@ internal class Die
...
 
-        //Property
+        public int Store { get; private set; }    
 
 
-        //Method
+        public int Roll()
+        {
+            Random random = new Random();
+            Store = random.Next(1, 7);
```
Comment (Nileso) [inline CMP1903_A1_2324/Die.cs]: hello

**Review by Nileso — COMMENTED** [review]
Very well articulated code

**Comment by Lemony-Juicy** [comment]
Very well articulated code

File: CMP1903_A1_2324/Game.cs [hunk 2]
```diff
@@ -15,7 +15,23 @@ internal class Game
...
+        {
+            Die die1 = new Die();
+            Die die2 = new Die();
+            Die die3 = new Die();
+
+            int roll1 = die1.Roll();
+            int roll2 = die2.Roll();
+            int roll3 = die3.Roll();
+
+            int total = roll1 + roll2 + roll3;
```
Comment (Nileso) [inline CMP1903_A1_2324/Game.cs]: nice code

Comment (naruto08060119487) [reply CMP1903_A1_2324/Game.cs]: THANK YOU

File: CMP1903_A1_2324/Testing.cs [hunk 3]
```diff
@@ -16,5 +17,28 @@ internal class Testing
...
+            Game game = new Game();
+            Console.WriteLine("Test 1");
+            string test1 = game.PLay();
+            Console.WriteLine(" ");
+            Console.WriteLine("Test 2");
+            string test2 = game.PLay();
+
+            Debug.Assert(test1 == test2, "Game test Failed");
+        }
+
```
Comment (Nileso) [inline CMP1903_A1_2324/Testing.cs:42]: good test

**Review by Nileso — APPROVED** [review]

**Review by naruto08060119487 — COMMENTED** [review]
CREATES A DIE

**Pull request closed without merging**